client/post-editor/editor-confirmation-sidebar/index.jsx

```diff
@@ -38,6 +39,21 @@ class EditorConfirmationSidebar extends React.Component {
 		this.props.onPublish( true );
 	};
 
+	getBusyButtonLabel( publishButtonStatus ) {
+		switch ( publishButtonStatus ) {
+			case 'update':
+				return this.props.translate( 'Updating…' );
+			case 'schedule':
+				return this.props.translate( 'Scheduling…' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **4** times:
`translate( 'Scheduling...' )` *ES Score: 25*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).